index.html

```diff
@@ -197,6 +197,13 @@ <h3>Beginner</h3>
                 <p>New to hackathons? No worries! You'll be provided with dedicated resources to help you be successful at Technica and beyond! This includes everything from an introduction to hackathons and Technica to universal coding and logic concepts!</p>
               </div>
             </div>
+            <div class="col-lg">
+              <div class="hardware" style="height: 100%">
+                <div></div>
+                <h3>Hardware</h3>
+                <p>Want to spark up your project with some hardware? Great news! At Technica, we’ll provide you with the resources for you to kickstart on your own hardware projects. You’ll have access to a kit as well as individualized sessions to help you master construction and coding fundamentals!</p>
```

Comment:
can you also add the hardware section to the mobile carousel below

Reply:
I'm a clown 🤦‍♀️